package.json

```diff
@@ -33,13 +33,14 @@
     "ramda": "~0.27",
     "vue": "~3",
     "vue-flatpickr-component": "~9",
-    "vue-i18n": "^9.2.2",
+    "vue-i18n": "^10.0.7",
```

Comment:
For some reason, I had trouble with Vue I18n until I updated the dependency. I'm not sure why that is. I definitely had i18n working the last time I was working on Vite things. In any case, it's probably a good thing to be on a more recent version of Vue I18n. This is the latest v10 version of Vue I18n. It's not easy to update to v11 yet: see getodk/central#1026.